python/instrumentation/openinference-instrumentation-bedrock/src/openinference/instrumentation/bedrock/_attribute_extractor.py

```diff
@@ -861,9 +862,35 @@ def extract_trace_id(cls, trace_data: dict[str, Any]) -> Any:
             if "traceId" in rationale:
                 return rationale["traceId"]
 
+        # For guardrail traces
+        if "traceId" in event_data:
+            return event_data["traceId"]
+
         # Generate a unique ID if none found
         return str(uuid.uuid4())
 
+    @classmethod
+    def get_attributes_from_guardrail_trace(cls, guardrail_trace: dict[str, Any]) -> dict[str, Any]:
+        """
+        Extract attributes from guardrail trace data.
+        """
+        guardrail_trace_data = {}
+
+        # Extract client_request_id from the guardrail metadata if present
+        metadata_attributes = cls.get_metadata_attributes(guardrail_trace.get("metadata", {}))
+        if client_request_id := metadata_attributes.get("client_request_id"):
+            guardrail_trace_data["client_request_id"] = client_request_id
+
+        # Normalize keys to snake_case for consistency in metadata
+        if "action" in guardrail_trace:
+            guardrail_trace_data["action"] = guardrail_trace["action"]
+        if "inputAssessments" in guardrail_trace:
+            guardrail_trace_data["input_assessments"] = guardrail_trace["inputAssessments"]
```

Comment:
do we have to also normalize the keys of the assessments themselves https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_GuardrailAssessment.html

inputAssessments
The details of the input assessments used in the Guardrail Trace.

Type: Array of [GuardrailAssessment](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_GuardrailAssessment.html) objects

outputAssessments
The details of the output assessments used in the Guardrail Trace.

Type: Array of [GuardrailAssessment](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_GuardrailAssessment.html) objects